pom.xml

```diff
@@ -92,6 +92,9 @@
              Force bigger region size in attempt to help G1 utilize heap fully. -->
         <air.test.jvm.additional-arguments>
             -XX:G1HeapRegionSize=32M
+            -XX:+UnlockDiagnosticVMOptions
+            -XX:GCLockerRetryAllocationCount=10
```

Comment:
only 10?

Reply:
It's 2 by default, I think 10 should be sufficient. Do you think it should be higher?

Reply:
oh, thought default is about 10. fine.
maybe add `<!-- bump from default of 2 -->` ?